cmd/runtimecfg/node-ip.go

```diff
@@ -83,6 +83,12 @@ func show(cmd *cobra.Command, args []string) error {
 }
 
 func set(cmd *cobra.Command, args []string) error {
+	// If primary ip address was already created, it means that nodeip-configuration has run already and no need to
+	// choose new ip, we should leave same configuration as we already set
+	if _, err := os.Stat(nodeIpFile); err == nil {
```

Comment:
Do we need to check for a zero length file or lack of read permissions?

`os.Stat` tends to be a poor choice, it seems better to test what we actual need, which is to read file that is not empty, or even better that contains a parseable IP.

Reply:
Sounds reasonable

Reply:
But lets see if this PR is relevant all, @cybertron ?

Reply:
So I think we agreed that we still need this one here, correct? So change this to reading file + parsing IP address?

Reply:
yep